receiver/tcpcheckreceiver/go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.23.6
 
 require (
 	github.com/google/go-cmp v0.6.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden v0.120.1
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden v0.121.0
```

Comment:
eh.. this seems wrong for the title of the PR

Reply:
it grafted on it

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38363 should go in first to remove those changes